controllers/toolchaincluster/toolchaincluster_controller_test.go

```diff
@@ -89,6 +90,7 @@ func TestClusterControllerChecks(t *testing.T) {
 
 		cl := test.NewFakeClient(t, stable, sec)
 		reset := setupCachedClusters(t, cl, stable)
+		stable.Status.Conditions = condition.AddOrUpdateStatusConditionsWithLastUpdatedTimestamp(stable.Status.Conditions, clusterReadyCondition())
```

Comment:
is this required?

Reply:
no i think i can set it while creating the cluster